bwrobot/Arrays[] Z
bwrobot/Arrays[] Z_old
bwrobot/Arrays[] mu_old
bwrobot/Arrays[] Su_old
bwrobot/Arrays[] Kaa_old
float64 kern_var
float64[] kern_lengthscale
float64 likelihood_var
float64[] xmean
float64[] ymean
int32 numloc
bwrobot/Arrays[] X
bwrobot/Arrays[] Y

